poetry/utils/inspector.py

```diff
@@ -49,7 +49,7 @@ def inspect_wheel(
             "version": "",
             "summary": "",
             "requires_python": None,
-            "requires_dist": None,
+            "requires_dist": [],
```

Comment:
Thanks for the quick PR last weekend! I've been using a version of this fix for the last week without issue

In the `inspect_sdist` method, the `requires_dist` key is type cast as a list ([line 104](https://github.com/sdispater/poetry/blob/2464c19196a89a109cbe2c810835f0e0bc896722/poetry/utils/inspector.py#L104)). For consistency, [line 73](https://github.com/sdispater/poetry/blob/2464c19196a89a109cbe2c810835f0e0bc896722/poetry/utils/inspector.py#L73) could be `info["requires_dist"] = list(meta.requires_dist)` instead of `info["requires_dist"] = meta.requires_dist`

Reply:
FYI: updated my prior comment. Also to add, in the case that `pkginfo.Wheel(str(file_path))` throws a ValueError, `meta` is undefined
https://github.com/sdispater/poetry/blob/2464c19196a89a109cbe2c810835f0e0bc896722/poetry/utils/inspector.py#L55-L60

Which would cause an error in a subsequent line

https://github.com/sdispater/poetry/blob/2464c19196a89a109cbe2c810835f0e0bc896722/poetry/utils/inspector.py#L70-L72

Which should probably be:

```py
if meta.requires_python:
    info["requires_python"] = meta.requires_python
```